apps/workspace-host/src/interface.js

```diff
@@ -251,6 +252,13 @@ async
         await dockerAttemptKillAndRemove(docker.getContainer(container.Id));
       }
     },
+    async () => {
+      cache.init({
```

Comment:
This function is async; you need to await the returned promise.

Reply:
Updated to await this promis in [eae6537](https://github.com/PrairieLearn/PrairieLearn/pull/9408/commits/eae65374a2f77e98f4937bcd4537406b04524243).